coordinator/migrations/2023-04-17-185626_create_users/up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE "users" (
+    id SERIAL PRIMARY KEY NOT NULL,
+    pubkey TEXT NOT NULL,
```

Comment:
We potentially could have gone with making the pubkey the id, but I guess it doesn't matter that much. :)